cmd/root.go

```diff
@@ -263,6 +265,21 @@ default values.`,
 		"",
 		"Data-dir configures the location of logs and data for validation. This will override the data_directory from configuration file",
 	)
+
+	checkDataCmd.Flags().Int64Var(
+		&tableSize,
+		"table-size",
+		0,
+		`Table-size configures the TableSize for badger DB. If table-size != 0, this will override the table_size from configuration file`,
```

Comment:
what is the default? add in here

Reply:
we have different defaultTableSize based on different mode, and we check the range in rosetta-cli/pkg/tester/data.go.

for here, 0 is out of the range, so I configure a default 0 here. which means if the person do not provide this field, we will use default value